### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by yayaismaail**
In this milestone I:

- Create tables in SQL.
- Insert and query data in SQL and
- Attached the screenshot of the results of the queries as the project requirements
vet1
vet2
vet3
vet4

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,13 @@
 /* Database schema to keep the structure of entire database. */
 
+-- CREATE TABLE animals (
+--     name varchar(100)
+-- );
```
Comment (V-Blaze) [inline schema.sql:5]: - [OPTIONAL] I'll suggest you totally remove these boiler-plate codes as leaving them as a comment makes your codebase look a bit messy and unprofessional :heavy_check_mark:

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,10 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+-- INSERT INTO animals (name) VALUES ('Luna');
+-- INSERT INTO animals (name) VALUES ('Daisy');
+-- INSERT INTO animals (name) VALUES ('Charlie');
```
Comment (V-Blaze) [inline data.sql:5]: - [OPTIONAL] I'll suggest you totally remove these boiler-plate codes as leaving them as a comment makes your codebase look a bit messy and unprofessional :heavy_check_mark:

**Review by V-Blaze — APPROVED** [review]
Hi @yayaismaail 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved-accepted


## Highlights

- Table created correctly and correct data inseted💯
-  Querry result screenshots added to pull requests👍
- All queries written correctly 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (yayaismaail) [reply schema.sql:5]: @V-Blaze, thanks for your suggestions

**Pull request merged**